scripts/ghutils.py

```diff
@@ -436,19 +435,32 @@ def create_release(repo, version, bodyfile):
     # Store version in context for use in notifications
     ctx_obj.version = version
     try:
-        release = repo.create_release(
-            tag_name=version,
-            name=version,
-            body=release_notes,
-        )
-        logger.info("Release {} created.".format(version))
+        try:
+            release = [r for r in repo.iter_releases() if r.name == version][0]
+            if release.body != release_notes:
+                raise ReleaseFailureException(
+                    "Release failed. Github release {version} already exists"
```

Comment:
At this point, is the tag guaranteed to be pointing to the expected hash or is it simply confirming that the tag names match?

Reply:
This just checks names, the tag creation function checks the tag SHAs and recreates if necessary.